lib/chef/application/solo.rb

```diff
@@ -207,7 +214,14 @@ def reconfigure
       FileUtils.mkdir_p(recipes_path)
       tarball_path = File.join(recipes_path, 'recipes.tgz')
       fetch_recipe_tarball(Chef::Config[:recipe_url], tarball_path)
-      Chef::Mixin::Command.run_command(:command => "tar zxvf #{tarball_path} -C #{recipes_path}")
+      if Chef::Config[:github]
+        cookbooks_dir = File.join(recipes_path, 'cookbooks')
+        FileUtils.rm_rf(cookbooks_dir, :secure => true)
```

Comment:
Can we do something other than `rm_rf`?!  that feels wrong and likely to cause someone to have a really bad day.